bitcoin/src/blockdata/locktime/absolute.rs

```diff
@@ -419,6 +419,9 @@ impl Height {
     /// The maximum absolute block height.
     pub const MAX: Self = Height(LOCK_TIME_THRESHOLD - 1);
 
+    /// Creates a new instance having its default value.
+    pub const fn new() -> Height { Height(0) }
```

Comment:
Perhaps  `Height::ZERO` would be slightly better.

Reply:
Oh yes, I just forget to correct them . My Bad